### DISCUSSION TITLE
Φυσική παρουσία για παρουσίαση προόδου δύο φορές

### DISCUSSION THREAD

**Discussion by Icelandan** [courses-ionio/sw#1143] [Q&A] [answered]
Υπάρχει προσωπικό θέμα που θα ήταν απρεπές να αναφερθεί στις συζητήσεις του github, και μου είναι αδύνατο να παραβρεθώ Κέρκυρα. Ζητάω από τον διδάσκοντα να ελέγξει το webmail του.

**Comment by epidrome** [comment, marked as answer]
* Η υποχρέωση φυσικής παρουσίας είναι μια επιλογή του Υπουργείου-Ιδρύματος και επομένως δεν υπάρχει καμία δυνατότητα για εξ αποστάσεως παρουσίαση, αν και εμείς θα το προτιμούσαμε αφού για τα μαθήματα μας πρακτικά δεν υπάρχει καμία διαφορά. 

* Η μόνη διευκόλυνση που μπορούμε να κάνουμε για τις δύο υποχρεωτικές συναντήσεις προόδου είναι να παρουσιάσετε σε άλλες ώρες-μέρες την πρόοδο σας στον συνδιδάσκοντα του μαθήματος με την υποχρέωση όμως να καταγράψετε τα σχόλια βελτίωσης που σας έδωσε σε μια διακριτή ενότητα στην αναφορά σας.
